pkg/kubelet/container/runtime.go

```diff
@@ -119,6 +119,8 @@ type Runtime interface {
 	// stream the log. Set 'follow' to false and specify the number of lines (e.g.
 	// "100" or "all") to tail the log.
 	GetContainerLogs(pod *api.Pod, containerID ContainerID, logOptions *api.PodLogOptions, stdout, stderr io.Writer) (err error)
+	// Remove a container. If the container is still running, an error is returned.
```

Comment:
It looks like `rkt` can already handle containerID like in [this](https://github.com/kubernetes/kubernetes/pull/28627/files#diff-a53fa76e941e0bdaee26dcbc435ad2ffR110) case. So it should be OK I think.